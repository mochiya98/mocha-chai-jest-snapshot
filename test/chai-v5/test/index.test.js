import { expect } from 'chai'

describe('chai-v5', () => {
  it('should work', () => {
    expect('chai v5').toMatchSnapshot()
  })

  it('should fail', () => {
    expect('chai v4').toMatchSnapshot()
  })
})
