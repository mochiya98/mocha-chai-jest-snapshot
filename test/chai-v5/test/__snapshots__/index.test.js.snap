// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`chai-v5 should work 1`] = `"chai v5"`;
